src/libraries/scroller.js

```diff
@@ -872,12 +872,10 @@ define(['browser', 'layoutManager', 'dom', 'focusManager', 'ResizeObserver', 'sc
         if (item === undefined) {
             this.slideTo(this._pos[location], immediate);
         } else {
-
-            //if (!transform) {
-
-            //    item.scrollIntoView();
-            //    return;
-            //}
+            if (!transform) {
+                item.scrollIntoView();
+                return;
+            }
```

Comment:
```
Uncaught ReferenceError: transform is not defined
    at scrollerFactory.to (scroller.js:875)
    at scrollerFactory.toCenter (scroller.js:921)
    at HTMLDivElement.<anonymous> (emby-scroller.js:14)
    at focus (focusManager.js:42)
    at Object.autoFocus (focusManager.js:22)
    at hometab.js:41
```
Because function is defined for prototype and var is out of scope.
Can be checked with TV layout.